File: java/workbookpivottable_refresh.java
HttpResponse<String> response = Unirest.post("https://graph.microsoft.com/%7Bver%7D/drive/root/workbook/worksheets/%7Bid%7D/pivotTables/%7Bid%7D/refresh")
  .header("content-type", "application/json")
  .body("{ }")
  .asString();

File: python/workbookpivottable_refresh.py
import http.client

conn = http.client.HTTPSConnection("graph.microsoft.com")

payload = "{ }"

headers = { 'content-type': "application/json" }

conn.request("POST", "/%7Bver%7D/drive/root/workbook/worksheets/%7Bid%7D/pivotTables/%7Bid%7D/refresh", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))
